src/scripts/crowdin-import.ts

```diff
@@ -0,0 +1,348 @@
+// Library requires
+const { copyFileSync, existsSync, mkdirSync, readdirSync } = require("fs")
+const { resolve, join } = require("path")
+const argv = require("minimist")(process.argv.slice(2))
+
+/**
+ * Console flags
+ * -v,--verbose    Prints verbose console logs
+ * -f,--full       Prints full name of buckets in summary
+ *
+ *
+ * Follow these steps to import translations from Crowdin export:
+ *
+ * 1. Copy languages folder from Crowdin export to ./.crowdin
+ *    ie. ./.crowdin/{lang-codes}
+ *    Tip: Run `yarn crowdin-clean` to initialize the `.crowdin` folder. Can
+ *    also be used to erase contents when finished.
+ *
+ * 2. Select buckets to import by adding the number of the corresponding
+ *    content bucket to the chosen language array below
+ *    ie. `es: [1, 10],` would import the "Homepage" and "Learn" buckets for Spanish
+ *
+ * 3. Save file without committing \*
+ *
+ * 4. Execute script by running `yarn crowdin-import`
+ *
+ * 5. If successful, copy `GATSBY_BUILD_LANGS={langs}` output and paste in
+ *    your `.env`, then build site to test results.
+ *
+ * Remember: Revert working changes to this file before committing Crowdin import
```

Comment:
(Not strong) idea:

Perhaps we could remove this temporary `USER_SELECTION` state (which conflicts with git) in favor of the `.crowdin` folder structure.

Instead of importing everything in the `.crowdin` folder, the user only needs to import the buckets they need, and the script will import everything it reads from that folder. So you would be moving this `USER_SELECTION` state to the folder itself.

--------
However, I prefer the current approach. I think its probably faster from an operational perspective.

Reply:
I like that idea. Would make the process faster.

I also think that the 3rd script (`cleanup`) is not necessary. Every time you call the `setup`, should regenerate the `.crowdin` folder for you keeping the hole process simpler and minimal.